### ISSUE TITLE
mat-card does not render correctly

### ISSUE THREAD

**Issue by allanbond** [hamzahamidi/Angular6-json-schema-form#51]
**Describe the bug**
which template:

* [X] MaterialDesignFrameworkModule — Material Design
* [ ] Bootstrap3FrameworkModule — Bootstrap 3
* [ ] Bootstrap4FrameworkModule — Bootstrap 4
* [ ] NoFrameworkModule — plain HTML
* [ ] Other (please specify below)
  
`"type": "card"` incorrectly overwrites the class attribute of `<mat-card>`. `<mat-card>` should have `class="mat-card"`, but it doesn't.

**To Reproduce**

```javascript
[
  {
    "type": "card",
    "title": "My Card",
    "items": [
      {
        "type": "div",
        "display": "flex",
        "flex-direction": "row",
        "options": {
          "fxLayoutGap": "10px"
        },
        "items": [
          {
            "key": "firstName",
            "notitle": true,
            "placeholder": "First Name",
            "options": {
              "floatLabel": "always",
              "fxFlex": "1 1 35%"
            }
          },
          {
            "key": "lastName",
            "notitle": true,
            "placeholder": "Last Name",
            "fxFlex": "1 1 30%",
            "options": {
              "floatLabel": "always"
            }
          }
       ]
   }
]
```

**Expected behavior**
When mat-card is inserted into the DOM, it should look something like this: `<mat-card class=".mat-card">`

**Desktop (please complete the following information):**

- OS: MacOS
- Browser: chrome
- Version: 70

**Context**
This prevents mat-card from rendering properly.

**Possible Implementation**
in flex-layout-section.component.ts

```html
    <mat-card *ngIf="containerType === 'card'"
      [class]="options?.htmlClass || ''"
```

should be

```html
    <mat-card *ngIf="containerType === 'card'"
      [ngClass]="options?.htmlClass || ''"
```

**Comment by hamzahamidi** [comment]
@allanbond Thank you for pointing the isue. Can you make a PR to fix this issue?

**Comment by kenisteward** [comment]
@hamzahamidi this has been fixed as #55 I can confirm as well
